src/dev/build/tasks/patch_native_modules_task.ts

```diff
@@ -66,12 +66,12 @@ const packages: Package[] = [
       // * upload to kibana-ci-proxy-cache bucket
       'linux-arm64': {
         url:
-          'https://storage.googleapis.com/kibana-ci-proxy-cache/node-re2/uhop/node-re2/releases/download/1.15.4/linux-arm64-72.gz',
-        sha256: '5942353ec9cf46a39199818d474f7af137cfbb1bc5727047fe22f31f36602a7e',
+          'https://storage.googleapis.com/kibana-ci-proxy-cache/node-re2/uhop/node-re2/releases/download/1.15.4/linux-arm64-83.gz',
+        sha256: '4eb524ca9a79dea9c07342e487fbe91591166fdbc022ae987104840df948a4e9',
       },
```

Comment:
Nit: the comment on line 65 above says this:

```
      // * gzip -c build/Release/re2.node > linux-arm64-72.gz
```

Can we change it accordingly?

Reply:
Fixed 👍